etna/core/blocks/image.py

```diff
@@ -9,12 +9,66 @@
 from etna.core.blocks.paragraph import APIRichTextBlock
 
 
+class APIImageChooserBlock(ImageChooserBlock):
+    """
+    APIImageChooserBlock inherits ImageChooserBlock and adds the ability
+    to generate different renditions for JPEG and WebP formats. Without
+    this, and the get_api_representation override, the API would return
+    the database entry - which is just an image ID. This would require
+    a second API call to get the image renditions.
+
+    rendition_size defaults to `fill-600x400`, but can be specified
+    when the block is used, e.g:
+    image = APIImageChooserBlock(rendition_size="original")
+
+    quality also defaults to 80, and can be specified in the same way
+    as rendition_size.
+    """
+
+    def __init__(
+        self,
+        required=True,
+        help_text=None,
+        rendition_size="fill-600x400",
+        quality=80,
+        **kwargs,
+    ):
+        self.quality = quality
+        self.rendition_size = rendition_size
+        super().__init__(required=required, help_text=help_text, **kwargs)
+
+    def get_api_representation(self, value, context=None):
+        jpeg_image = value.get_rendition(
+            f"{self.rendition_size}|format-jpeg|jpegquality-{self.quality}"
+        )
+        webp_image = value.get_rendition(
+            f"{self.rendition_size}|format-webp|webpquality-{self.quality}"
+        )
+
+        return {
+            "id": value.id,
+            "title": value.title,
+            "image_jpeg": {
+                "url": jpeg_image.url,
+                "full_url": jpeg_image.full_url,
+                "width": jpeg_image.width,
+                "height": jpeg_image.height,
+            },
+            "image_webp": {
+                "url": webp_image.url,
+                "full_url": webp_image.full_url,
+                "width": webp_image.width,
+                "height": webp_image.height,
+            },
+        }
+
+
 class ImageBlock(blocks.StructBlock):
     """
     An image block which allows editors to ensure accessibility is reflected on the page.
     """
 
-    image = ImageChooserBlock(required=False)
+    image = APIImageChooserBlock(required=False)
```

Comment:
While we're here, should `required` by `true`?

Reply:
Good point... Not sure why it isn't!